### PR TITLE
Plugins: Copy update for private site eligibility hold

### PR DESCRIPTION
**Pull request by roundhill**
Updates the copy for the private site eligibility hold to include `or hidden`, since this is another option the user can choose to enable the transfer.

### CODE REVIEW

File: client/blocks/eligibility-warnings/hold-list.jsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ function getHoldMessages( siteSlug, translate ) {
 		},
 		SITE_PRIVATE: {
 			title: translate( 'Private site not supported' ),
-			description: translate( 'Make your site public to resolve.' ),
+			description: translate( 'Make your site public or hidden to resolve.' ),
```
Comment (a8ci18n) [inline client/blocks/eligibility-warnings/hold-list.jsx]: :disappointed: 17 existing translations will be lost with this change.

Comment (yoavf) [inline client/blocks/eligibility-warnings/hold-list.jsx]: Drive-by comment: I think this string isn't clear enough. "To resolve"? What does that mean exactly here?
